pkgs/applications/window-managers/weston/default.nix

```diff
@@ -1,12 +1,16 @@
 { lib, stdenv, fetchurl
 , meson, ninja, pkg-config, python3, wayland-scanner
-, cairo, dbus, lcms2, libdrm, libevdev, libinput, libjpeg, seatd, libxkbcommon
-, mesa, wayland, wayland-protocols
-, pipewire ? null, pango ? null, freerdp ? null
-, libXcursor ? null, libva ? null, libwebp ? null, xwayland ? null
-# beware of null defaults, as the parameters *are* supplied by callPackage by default
-, buildDemo ? true
-, buildRemoting ? true, gst_all_1
+, cairo, dbus, lcms2, libdrm, libevdev, libinput, libjpeg, libxkbcommon, mesa
+, seatd, wayland, wayland-protocols
```

Comment:
An example of a change (swapping mesa and seatd) that makes the diffs unnecessarily difficult to read IMO.